models.py

```diff
@@ -104,15 +90,67 @@ class DeckCards(db.Model):
     __tablename__ = "deck_cards"
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
     deck_id = db.Column(db.Integer, db.ForeignKey("decks.id", ondelete="cascade"))
-    card_id = db.Column(db.Integer, nullable=False)
+    cards_info = db.relationship("Cards")
     card_qty = db.Column(db.Integer, nullable=False)
-    card_name = db.Column(db.Text, nullable=False)
-    card_img = db.Column(db.Text, nullable=False)
-    card_colors = db.Column(db.Text)
-    card_type = db.Column(db.Text, nullable=False)
-    card_cmc = db.Column(db.Integer)
-    card_power = db.Column(db.Integer)
-    card_toughness = db.Column(db.Integer)
+
+
+class CardConnection(db.Model):
+    __tablename__ = "card_connection"
+    card_multiverse_id = db.Column(
+        db.Integer, db.ForeignKey("card_info.card_multiverse_id"), primary_key=True
+    )
+    cards_owned = db.Column(
```

Comment:
This comment is for lines 102, 105, and 108. Assuming this is a join table, what do these columns represent? Do they correspond to an ID on another table? If so, maybe we could give them a name like `wish_list_id`, or something similar, for readability purposes.